CHANGELOG.md

```diff
@@ -1,212 +1,211 @@
-- All EMS alerts include an impact annotation. Thanks to @Divya for raising.
-
-- :ear_of_rice: Harvest includes new templates to collect:
-  - Network filesystem (NFS) rewinds performance metrics (rw_ctx). Thanks to @shawnahall71 for raising
-  - Network data management protocol (NDMP) session metrics. Thanks to @schumijo for raising.
-
-- :closed_book: Documentation additions
-  - Harvest describe why and how to configure Docker's logging drivers [Docker logging configuration](https://netapp.github.io/harvest/nightly/install/containers/#note-on-docker-logging-configuration) Thanks to @Madaan for raising.
-  - How to create templates that use ONTAP's private CLI [details](https://netapp.github.io/harvest/nightly/configure-rest/#ontap-private-cli)
-  - How to create custom Grafana dashboards [Steps](https://netapp.github.io/harvest/nightly/dashboards/#creating-a-custom-grafana-dashboard-with-harvest-metrics-stored-in-prometheus)
-  - How to validate your `harvest.yml` file and share a redacted copy with the Harvest team. [Details](https://netapp.github.io/harvest/nightly/help/config-collection/)
-  - Harvest describes high-level concepts [here](https://netapp.github.io/harvest/nightly/concepts/) Thanks to @norespers for raising.
-
-- All constituents are disabled by default for workload detail performance templates.
-- The `bin/harvest zapi` CLI now supports a `timeout` argument.
-- Harvest performance collectors (ZapiPerf and RestPerf) ask ONTAP for performance counter metadata every 24 hours instead of every 20 minutes. Thanks to BrianMa for raising.
-- The Harvest REST collector's `api_time` metric now includes the API time for all template endpoints. Thanks to ChristopherWilcox for raising. 
+  - Adaptive QoS used percentage tracking in Workload dashboard. Thanks to @faguayot for reporting.
+  - Network dashboard now includes ethernet errors. Thanks to Rusty Brown for contributing.
+  - Volume growth rate panels in Volume dashboard. Thanks to AlessandroN for reporting.
+  - BMC firmware version in Node dashboard. Thanks to @summertony15 for reporting.
+  - I/O density panels in Volume dashboard. Thanks to @jgasher for reporting.
+  - Forecasting Volume capacity in Volume dashboard. Thanks to @s-kuchi for reporting.
+  - SVM dashboard now includes NFS4.2 panels. Thanks to Didlier for reporting.
+
+- :ear_of_rice: Harvest includes a new template to collect lock counts at node, SVM, LIF, and volume level. Thanks to @troysmullerna for reporting.
+
+- :closed_book: Documentation Additions:
+  - How to customize Prometheus retention period in Docker instructions. Thanks to @WayneShen2 for the suggestion.
+  - How to use endpoints in REST Collector template. Thanks to Hubert for reporting.
+  - EMS Alert [runbook](https://netapp.github.io/harvest/nightly/resources/ems-alert-runbook/).
+  - How to use `confpath` to extend templates.
+
+- Harvest supports embedded exporters in Harvest configuration. Thanks to @wagneradrian92 for reporting.
```

Comment:
Harvest supports embedded exporters in Harvest configuration. This means you can define your exporters in one place instead of multiple.